README.md

```diff
@@ -6,10 +6,13 @@ As of version 0.1.0 it also supports creating archives.
 It's handy if you do not have shell access. E.g. if you want to upload a lot of files (php framework or image collection) as archive - because it is much faster than uploading each file by itself.
 
 
-## Requirements    
+## Requirements
 PHP 5.3 and newer
 (If you still run PHP < 5.6 you should consider updating PHP. These old PHP versions do not get any security updates and your site may be vulnerable.)
 
+## Security
+* Don't leave utility on production server. It can easily be used to extract sensitive information (like configuration files with login/password to your database server). Delete it right after usage.
```

Comment:
Exactly this. Also the script contains a few vulnerabilities so it's not recommended to leave it accessible for everyone on a webserver.

This section should be added to the readme.